staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -150,13 +152,14 @@ func logPanic(ctx context.Context, r interface{}) {
 var ErrorHandlers = []ErrorHandler{
 	logError,
 	func(_ context.Context, _ error, _ string, _ ...interface{}) {
```

Comment:
stepping back... I'm super confused about how this will ever do anything more than sleep for about 1ms ... every HandleError call invokes this method, and every invocation creates a *new* rudimentaryErrorBackoff instance

we never save this `backoff` instance anywhere so we can reuse it... how is `lastErrorTime` ever used more than once?

Reply:
looks like this was broken in https://github.com/kubernetes/kubernetes/commit/5a130d2b71e5d70cfff15087f4d521c6b68fb01e#r156471569

Reply:
cc @pohly for visibility ... need to be super careful refactors around context threading aren't breaking

Reply:
Ack, sorry for that. I see the problem now. Thanks for taking care of it!